inspec-core.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "license-acceptance", ">= 0.2.13", "< 2.0"
   spec.add_dependency "thor",               ">= 0.20", "< 2.0"
   spec.add_dependency "json_schemer",       "~> 0.2.1"
-  spec.add_dependency "method_source",      "~> 0.8"
```

Comment:
you might want to make this >= 0.8 and < 2 so we can pull in any known working version